### PR TITLE
Int.div bug fix

### LINKED ISSUE #192: [bug] Wrong int.div implementation
The current `int.div` attribute implementation is incorrect.  
```
# Divide this one by another int
  [x] > div
    mul (x.pow -1) > @
```
where `pow` is defined as:  
```java
/**
 * POW.
 *
 * @since 1.0
 */
public class EOint$EOpow extends PhDefault {

    public EOint$EOpow(final Phi parent) {
        super(parent);
        this.add("x", new AtFree());
        this.add("φ", new AtBound(new AtLambda(this, self -> new Data.ToPhi(
            (long) Math.pow(
                new Data.Take(self.attr("ρ").get()).take(Long.class),
                new Data.Take(self.attr("x").get()).take(Long.class)
            )
        ))));
    }

}
```
Hence, the assumption here is such that `a / x = a * x^(-1)` which is totally wrong in the integer domain (since `x^(-1) = 0` for all integers except `0` and `1`).  
**Example**  
```
+package sandbox
+alias sprintf org.eolang.txt.sprintf
+alias stdout org.eolang.io.stdout

[args...] > app
  stdout > @
    sprintf
      "%d\n%d\n"
      2.pow -1
      2.div 2

```
**Running**  
```
IN$: ./run.sh
OUT>: 0
OUT>: 0
IN$: 
```  
The correct result here, surely, is `1`.  
It could be algebraically concluded, that the `a. div b`  works as described:  
1. `a` if `b = 1`  
2. product of `a` and the large int number (i.e. infinity) if `b = 0`
3. `0` if  `b != 0 && b != 1 `

### PR DESCRIPTION
**Pull request by IngeniariusSoftware**
Fixes #192

### CODE REVIEW

**Comment by IngeniariusSoftware** [comment]
@yegor256 ready for review

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#209](https://github.com/cqfn/eo/issues/209); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: eo-runtime/src/test/eo/org/eolang/int-tests.eo [hunk 1]
```diff
@@ -43,3 +43,36 @@
         n.eq 3
         minus1.add minus2
         rec (n.sub 1) (minus1.add minus2) minus1
+
+# Checks that division by zero returns an error object
+[] > zeroDivision
+  eq. > @
+    (2.div 0).msg
+    (error "Division by zero is undefined").msg
```
Comment (yegor256) [inline eo-runtime/src/test/eo/org/eolang/int-tests.eo:51]: @IngeniariusSoftware why not just 

```
"Division by zero is undefined"
```

**Comment by IngeniariusSoftware** [comment]
@yegor256, sorry, accidentally resolved the comment. For explicit error indication

File: eo-runtime/src/test/eo/org/eolang/int-tests.eo (hunk 1 above)
Comment (IngeniariusSoftware) [reply eo-runtime/src/test/eo/org/eolang/int-tests.eo:51]: For explicit error indication

Comment (yegor256) [reply eo-runtime/src/test/eo/org/eolang/int-tests.eo:51]: @IngeniariusSoftware looks weird. I would just compare with a string.

Comment (IngeniariusSoftware) [reply eo-runtime/src/test/eo/org/eolang/int-tests.eo]: @yegor256 no problem, fixed it

**Comment by yegor256** [comment]
@IngeniariusSoftware what's up with this? ready?

**Comment by IngeniariusSoftware** [comment]
@yegor256 ready

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:cqfn/eo#209` is not assigned, can't get performer

**Pull request merged**